src/mesh/X3D_Draco_Mesh_Reader.i.hh

```diff
@@ -0,0 +1,98 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   mesh/X3D_Draco_Mesh_Reader.i.hh
+ * \author Ryan Wollaeger <wollaeger@lanl.gov>, Kendra Keady
+ * \date   Thursday, Jul 12, 2018, 08:46 am
+ * \brief  X3D_Draco_Mesh_Reader class implementation header file.
+ * \note   Copyright (C) 2018 Los Alamos National Security, LLC.
+ *         All rights reserved. */
+//---------------------------------------------------------------------------//
+
+#include "ds++/DracoStrings.hh"
+
+namespace rtt_mesh {
+
+//---------------------------------------------------------------------------//
+// PRIVATE FUNCTIONS
+//---------------------------------------------------------------------------//
+/*!
+ * \brief Generate a map from a block in an x3d file
+ *
+ * \param[in] block_name name of parsed x3d block
+ *
+ * \return map of mesh data with key of type KT and value of type VT
+ */
+template <typename KT, typename VT>
+std::map<KT, std::vector<VT>>
+X3D_Draco_Mesh_Reader::map_x3d_block(std::string block_name, int &dist) {
+
```

Comment:
Can `block_name` be `const &`?

Reply:
Thank you for finding this - the `block_name` is now `const &`